Initial Span Smart Breaker Panel support as Power Meter (#86)
Enapter/marketplace: power_meters/span_io_panel/README.md

# SPAN.io Smart Panel

> This [Enapter Device Blueprint](https://go.enapter.com/marketplace-readme) integrates **SPAN.IO Smart Panels** - breaker panel. The connection is via standard http REST calls.

Reviewer: ```suggestion
This [Enapter Device Blueprint](https://go.enapter.com/marketplace-readme) integrates **SPAN.IO Smart Panel** - breaker panel. The connection is set via standard HTTP REST calls.
```

## Connect to Enapter

- Sign up to the Enapter Cloud using the [Web](https://cloud.enapter.com/) or mobile app ([iOS](https://apps.apple.com/app/id1388329910), [Android](https://play.google.com/store/apps/details?id=com.enapter&hl=en)).
- Use the [Enapter Gateway](https://handbook.enapter.com/software/gateway/2.0.0/setup/) to run the Virtual UCM.
- Create the [Enapter Virtual UCM](https://handbook.enapter.com/software/software.html#%F0%9F%92%8E-virtual-ucm).
- [Upload](https://developers.enapter.com/docs/tutorial/uploading-blueprint/) this blueprint to ENP-VIRTUAL.
- Please ensure that your installer is connecting the LAN connection of the battery to your network.

> - Use the `Configure` command in the Enapter mobile or Web app to set up the LG RESU 10h/16h Prime:

Reviewer: ```suggestion
- Use the `Configure` command in the Enapter mobile or Web app to set up the SPAN.IO Smart Panel:
```

  - IP address (use either static IP or DHCP reservation);

## References

> - [SPAN.IO](https://span.io)

Reviewer: ```suggestion
- [SPAN.IO Smart Panel product page](https://www.span.io/panel)
```
